examples/websocket_client.py

```
@@ -0,0 +1,281 @@
+            }
+            
+            await self.websocket.send(json.dumps(join_message))
+            
+            # Wait for join confirmation
+            response = await self.websocket.recv()
+            response_data = json.loads(response)
+            
+            if response_data.get("event") == "phx_reply" and response_data.get("payload", {}).get("status") == "ok":
+                payload = response_data["payload"]["response"]
+                self.subscription_id = payload.get("subscription_id")
+                logger.info(f"✅ Connected to WandererKills WebSocket")
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)